src/python/library/build_wheel.py

```diff
@@ -194,6 +201,21 @@ def sed(pattern, replace, source, dest=None):
             if not os.path.exists(os.path.join(FLAGS.whl_dir, "perf_client")):
                 os.symlink("perf_analyzer", os.path.join(FLAGS.whl_dir, "perf_client"))
 
+    if FLAGS.windows:
+        cpdir(
+            "tritonclient/utils/shared_memory",
+            os.path.join(FLAGS.whl_dir, "tritonclient/utils/shared_memory"),
+        )
+        shutil.copyfile(
+            "tritonclient/utils/Release/cshm.dll",
+            os.path.join(FLAGS.whl_dir, "tritonclient/utils/shared_memory/cshm.dll"),
+        )
+        # FIXME: Enable when Windows supports GPU tensors
```

Comment:
Can you create a jira ticket for cuda shared memory support on windows if not already did so, and mention that here?

Reply:
I see that cuda-python is available on windows machine: https://pypi.org/project/cuda-python/#files
I don't think it would be too much work to support cuda shared memory for windows in Triton.

Reply:
That is going to be my next ticket, but I'll add the ticket ID to the comment.